### PR TITLE
add queries from staking app

### PR DESCRIPTION
**Pull request by KillerByte**
all changes:
* add queries from staking app
* have queries use codegen to link query function dependencies
* fix internal dependencies by lerna to use exact version (otherwise it uses the highest hash)
* fix modules to always export `module` definition
* add dependencies for `@synthetixio/data` for new queries

### CODE REVIEW

File: packages/queries/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/** @type {import('@ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+	preset: 'ts-jest',
+	testEnvironment: 'node',
+	coverageThreshold: {
+		global: {
+			branches: 0,
```
Comment (dvd-schwrtz) [inline packages/queries/jest.config.js:7]: do we need the config if thresholds are 0?

File: package.json [hunk 2]
```diff
@@ -50,5 +51,10 @@
 		"hooks": {
 			"pre-commit": "npm run lint"
 		}
+	},
```
Comment (clementbalestrat) [inline package.json:54]: what's an optional dependency?

File: packages/data/queries/rateUpdates/parse.ts [hunk 3]
```diff
@@ -8,7 +7,8 @@ export const parseRates = (rate: RateUpdate, networkId?: number): RateUpdate =>
 	const parsedRate =
 		networkId && networkId === NetworkId['Kovan-Ovm']
 			? rateValue
-			: ethers.utils.formatEther(rateValue);
+			: // todo: I have no idea what this code is supposed to do
+			  (rateValue / 1e18).toString();
```
Comment (clementbalestrat) [inline packages/data/queries/rateUpdates/parse.ts:11]: let's just get rid of it

File: packages/queries/src/constants.ts [hunk 4]
```diff
@@ -71,3 +71,5 @@ export const PERIOD_LABELS_MAP: Record<Period, PeriodLabel> = {
 };
 
 export const PERIOD_LABELS = Object.values(PERIOD_LABELS_MAP);
+
+export const COUNCIL_NOMINATIONS_URL = 'https://';
```
Comment (clementbalestrat) [inline packages/queries/src/constants.ts:75]: whats this URL for exactly?

**Review by clementbalestrat — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (KillerByte) [reply package.json:54]: npm now complains if its not included. maybe newer version or somethign? I think its a dependency that, if someone were to include the repository root (which they wouldn't), they can "optionally" install the dependency.

it did have a positive effect, not just on the pipeline, but also on my computer running pre-commit hooks, so its worthwhile to include them.

File: packages/data/queries/rateUpdates/parse.ts (hunk 3 above)
Comment (KillerByte) [reply packages/data/queries/rateUpdates/parse.ts:11]: can't becuase the test will fail otherwise

Comment (KillerByte) [reply packages/data/queries/rateUpdates/parse.ts:11]: we are probably going to replace the `@synthetixio/data` repo with a code generator in the near future so I'm not super keen on carefully maintaining this for the time being.

File: packages/queries/src/constants.ts (hunk 4 above)
Comment (KillerByte) [reply packages/queries/src/constants.ts:75]: its depended upon by one of the queries. Let me check what the actual value for that is supposed to be and fill it in.

**Pull request merged**